source/administration/changelog.md

```diff
@@ -4,6 +4,97 @@ This changelog summarizes updates to [Mattermost Team Edition](https://mattermos
+ - Removed references to ``SqlLite3`` from the code.
+ - Bleve updates are now logged in the config only when there is an actual change in the ``BleveSettings`` instead of on every config update.
+ - Profiling the Mattermost server with pprof is now available for Team Edition.
+ - Added attributes to split.io feature flags.
+
+### Bug Fixes
+ - Fixed unsafe access of properties of the plugin environment during ``ServePluginPublicRequest``.
+ - Fixed an issue where the Admin Console > Server Logs did not focus to the sidebar filter upon reload.
+ - Fixed an issue where the Gif picker appeared empty instead of showing a “No results” modal when no results were displayed.
+ - Fixed an issue where the keyboard accessibility controller was not allowed to resume left-hand side scroll after drag and drop.
+ - Fixed an issue where markdown links rendered incorrectly.
+ - Fixed an issue where the slack theme import failed due to changes in formatting of Slack export color schemes.
+ - Fixed an issue where tooltips were missing for channels with a long name.
+ - Fixed a race condition which would crash the app server due to improper handling of websocket closing.
+ - Fixed an issue where the PDF zoom failed to respond to zoom in/out/reset actions until the user scrolled.
+ - Fixed an issue where in a reply thread with the right-hand side expanded, attachments in a post draft got hidden behind the center channel text box.
+ - Fixed bugs related to replication lag for Enterprise Edition instances configured to use read replicas.
+ - Fixed an issue where Compliance Report field headers were not correctly aligned.
+ - Fixed an issue where the ``/join`` command was case-sensitive.
+ - Fixed an issue where one-character sidebar category names were not displayed.
+ - Fixed an issue with a theme discrepancy on close buttons on some modals in the System Console (when using a custom team theme).
+ - Fixed an issue where long text input in the right-hand pane was jumpy when selected.
+ - Fixed an issue where the Zoom level persisted across multi-attachment PDF previews.
+ - Fixed an issue where long image names pushed the show/hide control off the right side of the window.
+
+### config.json
+Multiple setting options were added to ``config.json``. Below is a list of the additions and their default values on install. The settings can be modified in ``config.json``, or the System Console when available.
+
+#### Changes to Team Edition and Enterprise Edition:
+ - Added ``EnableFileSearch``, ``ExtractContent``, ``ArchiveRecursion`` for file search. **XXXX ??**
+ - Added ``ExportSettings`` to add support for compressed export files with attachments.
+
+### Go Version
+ - v5.34 is built with Go ``1.15.5``.
+
+### Open Source Components
+ - Removed ``core-js`` from https://github.com/mattermost/mattermost-mobile.
+
+### API Changes
+ - Added API for files search ``/teams/{team_id}/files/search``. **XXXX ??**
+
```

Comment:
@esethna Do I keep the file search related changes here, or remove?

Reply:
Remove them from here. We should doc them with v5.35